modules/logging-loki-log-access.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_content-type: CONCEPT
+[id="logging-loki-log-access_{context}"]
+= Configuring Loki log access
+In {logging} 5.8 and later versions, you can configure fine grain access to logs using cluster wide policies, as well as by namespace. The ClusterLogging Operator does not grant all users access to logs by default. As an administrator, you may configure your users access as needed.
+
+ClusterRoles grant access to different types of logs:
+
+* `cluster-logging-application-view` grants permissions to read application logs.
+* `cluster-logging-infrastructure-view` grants permissions to read infrastructure logs.
+* `cluster-logging-audit-view` grants permission to read audit logs.
+* `logging-application-logs-reader` - Provides backwards compatibility with prior versions of logging.
+
+If you have upgraded to version 5.8, any authenticated user has permission to read logs in their namespaces. To change this delete the ClusterRole `logging-application-logs-reader` and the ClusterRoleBinding `logging-all-authenticated-application-logs-reader`.
+
+Users who are members of any group specified in the `adminGroups` field of the LokiStack CR are considered admins. Admin users have access to all application logs in all namespaces.
```

Comment:
"the all application logs in all namespaces" only applies if the user also has the required Kubernetes permission for the application tenant. For some of these groups this always applies (for example `system:cluster-admins` and `cluster-admin` usually have _all privileges_ by design).

```suggestion
Users who are members of any group specified in the `adminGroups` field of the LokiStack CR are considered admins. Admin users have access to all application logs in all namespaces, if they also get assigned the `cluster-logging-application-view` role.
```